cmd/telliot/main.go

```diff
@@ -30,34 +30,40 @@ func parseConfig(path string) (*config.Config, error) {
 	return cfg, nil
 }
 
-func createTellorVariables(ctx context.Context, logger log.Logger, cfg *config.Config) (contracts.ETHClient, *contracts.ITellor, *rpc.Account, error) {
+func createTellorVariables(ctx context.Context, logger log.Logger, cfg *config.Config) (contracts.ETHClient, contracts.ETHClient, *contracts.ITellor, []*rpc.Account, error) {
 
 	// Create an rpc client
 	client, err := rpc.NewClient(logger, cfg, os.Getenv(config.NodeURLEnvName))
 	if err != nil {
-		return nil, nil, nil, errors.Wrap(err, "create rpc client instance")
+		return nil, nil, nil, nil, errors.Wrap(err, "create rpc client instance")
+	}
+
+	// Create an websocker client for watch events.
+	clientWs, err := rpc.NewClient(logger, cfg, os.Getenv(config.NodeWSURLEnvName))
+	if err != nil {
+		return nil, nil, nil, nil, errors.Wrap(err, "create rpc client instance")
```

Comment:
I think lets keep it simple with just a single client.
And somewhere check if the connection supports subscription and return an error if it doesn't so that the users know they need to use a node that support listening for events.

Reply:
.